.travis.yml

```diff
@@ -0,0 +1,82 @@
+language: python
+python:
+- '3.4'
+cache: pip
+install:
+- pip3 install https://github.com/jnewland/home-assistant/zipball/rancho-de-newland
+script:
+- openssl req -x509 -newkey rsa:1024 -subj '/CN=example.com' -nodes -keyout /tmp/key.pem
+  -out /tmp/cert.pem
+- hass -c . --script check_config | tee /tmp/log
+- test "$(grep -c ERROR /tmp/log)" -eq 0
```

Comment:
This seems to be required, as `hass -c . --script check_config` exits 0 even if there are errors.